plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TypeConverter.java

```diff
@@ -108,6 +108,8 @@ public static Type toTrinoType(org.apache.iceberg.types.Type type, TypeManager t
             case VARIANT:
                 // TODO https://github.com/trinodb/trino/issues/24538 Support variant type
                 break;
+            case GEOMETRY:
```

Comment:
Do we plan to support Geometry/Geography types? Add todo with a github issue link